package.json

```diff
@@ -30,26 +32,45 @@
   "scripts": {
     "start": "stripes serve",
     "lint": "eslint .",
-    "test": "echo 'placeholder. no tests implemented'"
+    "test": "stripes test karma"
   },
   "devDependencies": {
+    "@folio/stripes-core": "^5.0.0",
+    "core-js": "^3.6.4",
+    "@folio/stripes-components": "~8.0.0",
+    "@folio/stripes-connect": "~6.0.0",
+    "@folio/stripes-cli": "^1.18.0",
+    "localforage": "^1.5.6",
+    "@bigtest/convergence": "^1.1.1",
+    "@bigtest/interactor": "^0.9.1",
+    "@bigtest/mocha": "^0.5.2",
+    "@bigtest/react": "^0.1.2",
     "@folio/eslint-config-stripes": "^5.1.0",
-    "@folio/stripes": "^4.0.0",
-    "@folio/stripes-cli": "^1.3.1",
+    "regenerator-runtime": "^0.13.3",
     "babel-eslint": "^10.0.3",
+    "chai": "^4.2.0",
     "eslint": "^6.2.1",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
-    "react-intl": "^4.5.3"
+    "mocha": "^8.0.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-intl": "^4.5.3",
+    "react-router": "^5.2.0",
+    "react-router-dom": "^5.2.0",
+    "sinon": "^7.2.2"
   },
   "dependencies": {
+    "@folio/stripes": "^4.0.0",
+    "miragejs": "^0.1.32",
+    "@bigtest/mirage": "^0.0.1",
     "@folio/react-intl-safe-html": "^2.0.0",
     "lodash": "^4.17.4",
     "prop-types": "^15.6.0"
   },
   "peerDependencies": {
     "@folio/stripes": "^4.0.0",
     "react": "*",
+    "react-dom": "*",
+    "react-router": "^5.2.0",
     "react-intl": "^4.5.3",
     "react-router-dom": "^4.3.1"
```

Comment:
`react-router` and `react-router-dom` need to stay in sync; the correct version here is `^5.2.0`.